Do not render multiple box-shadow sections

micahwood opened the pull request:
When using  the format ` type: 'html'` with a box-shadow token, two box-shadow sections are rendered. This pull request removes the duplicate render call and removes the extra blank space from test snapshot

aputinski commented:
I didn't create this particular formatter, so maybe I'm confused, but it looks like `box-shadow` is still there twice and what you removed was `drop-shadow` which was a third `box-shadow`?

https://github.com/micahwood/theo/blob/1d93f7ee3cc881af6e14e2b2aa98dbc8b0d2999a/lib/formats/html.js#L438

https://github.com/micahwood/theo/blob/1d93f7ee3cc881af6e14e2b2aa98dbc8b0d2999a/lib/formats/html.js#L444

micahwood commented:
Whoops, you are correct I just deleted the wrong line. Not sure how I managed to mess up a one line change. Fixed the commit and pushed it again so it should now be deleting the duplicate box-shadow render call.
